lib/r/extract_cluster.dart

```diff
@@ -54,6 +54,9 @@ String _basicTemplate(
   } else if (type == 'Hierarchical') {
     hd = 'Summary of the Hierarchical Cluster Analysis';
     md = "(built using 'Hierarchical'):";
+  } else if (type == 'BiCluster') {
```

Comment:
Peobably best to use lowercase throughout - otherwise the programmer will be confused as to which way you have decide to go.